File: class_4/class_4/app/src/main/java/com/alejouribesanchez/class_4/Main7Activity.java
package com.alejouribesanchez.class_4;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.RelativeLayout;
import android.widget.TableLayout;

public class Main7Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main7);

        TableLayout tableLayout = (TableLayout) findViewById(R.id.activity_main7);
        tableLayout.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(Main7Activity.this, Main8Activity.class);
                startActivity(intent);
            }
        });
    }
}

File: class_4/class_4/app/src/main/java/com/alejouribesanchez/class_4/Main7Activity.kt
package com.alejouribesanchez.class_4

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.RelativeLayout
import android.widget.TableLayout

class Main7Activity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main7)

        val tableLayout = findViewById(R.id.activity_main7) as TableLayout
        tableLayout.setOnClickListener {
            val intent = Intent(this@Main7Activity, Main8Activity::class.java)
            startActivity(intent)
        }
    }
}
